src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.ts

```diff
@@ -28,4 +30,8 @@ export class DashboardHelpComponent implements OnInit {
   openAboutModal() {
     this.modalRef = this.modalService.show(AboutComponent, null, { size: 'lg' });
   }
+
+  openFeedbackModal() {
+    this.bsModalRef = this.modalService.show(FeedbackComponent);
```

Comment:
?
```suggestion
    this.bsModalRef = this.modalService.show(FeedbackComponent, null, { size: 'lg' });
```

Reply:
> ?

https://github.com/ceph/ceph/pull/42603#discussion_r759194084

Reply:
I am sorry but I couldn't see anything in that link

Reply:
I'm pointing again...https://github.com/ceph/ceph/pull/42603#discussion_r759194084